### ISSUE TITLE
Add API backup URLs from the control panel

### ISSUE THREAD

**Issue by gl0vi** [UnamSanctam/UnamWebPanel#207]
I have a some miners in your control panel. Can I add backup API URLs from the panel? And remote control links like pastebin?
I tried creating this configuration from the panel and putting it to a miner but I can't see it in the new panel:
`{
    "remote-config": "https://pastebin.com/raw/XXXXXXXX,https://pastebin.com/raw/XXXXXXXX",
    "api-endpoint": "https://xxx.com/api/endpoint.php,https://xxx.com/api/endpoint.php",
}`

**Comment by UnamSanctam** [comment]
Yes that works, but you need to specify a pool, wallet and such as well in the configuration. That's so that the miner can verify that the call is correct.
